src/services/firebase/firestore.ts

```diff
@@ -270,19 +272,27 @@ export class FirestoreServices {
    */
   sendMessage = async (message: MessageProps) => {
     if (!this.conversationId) {
-      console.error(
+      throw new Error(
```

Comment:
if we throw error without try/catch, the app will crash

Reply:
ok I remove it